scripts/offline-bundle/image-bundle/import.sh

```diff
@@ -4,10 +4,7 @@ IFS=$'\n\t'
 
```

Comment:
It seems to me, this is all part of the `offline-bundle` that existed and was needed during `stackrox.io` times. Now, as we sunset stackrox.io image registries, and we don't create offline bundle in CI, we can remove offline-bundle stuff completely, I think.
See related https://srox.slack.com/archives/C3W47R5LP/p1651081299957489?thread_ts=1651079204.647909&cid=C3W47R5LP

That said, it shouldn't be a part of this PR and doesn't have to be done by you. The change you have here looks sane, therefore feel free to treat this comment as no-op.